charts/neo4jv5-internal/values.prod.yaml

```diff
@@ -38,6 +38,10 @@ neo4j:
           "name": "crunchbase-2.0.0",
           "neo4j_version": "5.3.0"
         },
+        {
+          "name": "cybersecurity-2.0.0",
+          "neo4j_version": "5.9.0"
+        },
```

Comment:
ready to merge on next maintenance prod slot
it has been tested on preprod with preview env.